### DISCUSSION TITLE
nodal interference consideration

### DISCUSSION THREAD

**Discussion by JeffBurris** [edwardkort/WWIDesigner#110] [NAF Study] [answered]
Hello and thanks for the opportunity to own and use such fine software for a price readily affordable to me during a relatively difficult time for me (free).

I was wondering if any anti nodal interference considerations were built into WID, or if we should supplement with something, perhaps when pushing limits of advisable hole constraints and such, like checking WID results with Mike Prairie's Nodal Interference Check spreadsheet?

Thank you

**Comment by edwardkort** [comment, marked as answer]
Jeff,

Your first stop for WIDesigner questions is the Wiki pages. For the NAF, in particular the page that links all the video tutorials I have made: [WIDesigner Video Tutorials](https://github.com/edwardkort/WWIDesigner/wiki/WIDesigner-Video-Tutorials).

Yes, WIDesigner fully implements nodal interference calculations (as defined by Mike Prairie's spreadsheet) in all the NAF Study optimizers. Watch video 10 from the page above.

Cheers,
Edward Kort
Wood Wind

**Reply by JeffBurris** [reply]
Hello Mr. Kort,
I thought I'd ask you via this existing thread instead of cluttering up your discussion area (albeit an originally embarrassing thread since, at the time, I overlooked the tutorial video entirely). 
I've watched the Nodal Interference video multiple times and I have a question.

I felt I understood what to do, but the Nodal Interference check has gone through a couple of updates since the version shown. The new one has a keen visual aid, and the field you advised creating to express that top hole ratio as a handy quotient has already been included for us "(L56-F14)/F11 = min. Hole 6 / Bore Length", but I'm wondering if the approach to using the spreadsheet has been changed at all (?).
It's no doubt a great update and I'll surely use it, but seemingly I'm at a catch-22 wanting to follow your original instructions for using it with WID.

My main concern:
This version requires I enter a value for the distance of the top hole from the foot. It's an input variable. Was that so for the older version?

I could just ignore it but it concerns me how it potentially affects everything else.

The whole point was that I was going to arrive there with nothing in hand but my bore length and bore size, and from there it would generate a Hole 6 Transition Value and a k2, I'd subtract the latter from the former, divide that by the bore length, and I'd have the quotient to put into constraints for the top hole ratio, Dimensionless field. 

Now, I believe I know the point of the new Nodal Interference Check version (Mr. Prairie's intent for the user) is to, oneself, play with prospective top hole distances from the foot, see what it does to the visual aid, and determine what's best oneself -- presumably "surfing" with one's top hole closely to the red line that represents the first high pressure zone's boundry (but not close enough to compromise it). 

I can do that; I'm just a wee bit confused, as I thought I understood from the tutorial I could have used the older spreadsheet to simply enter my bore length (from WID after Fipple Factor optimization) and I.D. (and fundamental note and temperature where I expect A4 to equal 440Hz etc.) and without my dealing with any measurements of the top hole to the foot I'd walk away with an ideal top hole ratio value for WID. I don't know my top hole position because I need the ratio to put into WID's constraints to determine the position of the top hole. Therein lies my perceived catch-22 if I had wanted to follow your exact instructions and not just use Mr. Prairie's visual aid.

I have attached the new spreadhseet version if you have any thoughts.
If you think it beneficial, I'd certainly save a copy of the old spreadsheet to my own records, for comparison (but I do not yet have a copy).
The new spreadsheet is no doubt superior (once I'm straightened out a bit) esp. since there is apparently a bug fix, but perhaps it might be good for learning purposes(?)
[Nodal Interference Check (NOV 2018).xlsx](https://github.com/edwardkort/WWIDesigner/files/6251864/Nodal.Interference.Check.NOV.2018.xlsx)
.
Thanks so much
Jeff

**Reply by edwardkort** [reply]
Hi Jeff,

I checked Mike's original spreadsheet (that I modified to add the top-hole ratio) and his latest spreadsheet (with that ratio already there): they gave identical answers for the top-hole/bore ratio. The ratio calculation does not use the measured Hole 6 position (F15 in the latest spreadsheet), so you don't need to put in a value.

Cheers,
Edward Kort
Wood Wind

**Comment by JeffBurris** [comment]
Thanks Mr. Kort. Forgive me for missing this in the video tutorials. I'd have been lost without them getting started and I can see I'm due to go back and finish them.

**Comment by JeffBurris** [comment]
Thanks very, very much!
I'm getting there. I'll bet I'm good to go for a while after noting one thing:

Part of the point of not just erroring on the excessively safe side against nodal interference (and going with .33 for all flutes or something) is to be in that sweet spot where you're safe from nodal interference but also not unnecessarily compromising either second register notes or cross-fingered notes (perhaps among other things).

In being very interested in producing a 1.4-octave friendly flute I see on the new spreadsheet the minimum hole size for 1.4 octaves for an A4 flute with .75 bore and .1875 walls hovers around .284". That's pretty darned large not only working with WID but any flutes I've played thus far which seemed to be just fine producing second registry tones (not that I held them up to a digital tuner to assess second registry deviations).

Do you have any thoughts on that  .28 to .29 inch minimum hole size whopper?
Are yours that big on such a flute? Not even my F4 flute with 1 inch diameter has that big of a smallest hole.

With gratitude for your time and thoughtful attention,
Jeff


________________________________

**Reply by edwardkort** [reply]
Jeff,

I put the same calculation in NAFlutomat 20 years ago. The equation comes from Lew Paxton Price. And I "religiously" follow it, but only for Hole 1. The other holes (except for maybe Hole 2, if your flute will play the major 10th) don't really contribute to the second-octave notes.

Personally, I like large holes. The flute sounds more open, not constipated. And a large Hole 1 better balances out the timbre change between the fundamental and the minor third. Plus it makes the half-holed major second voice better.

Cheers,
Edward

**Comment by JeffBurris** [comment]
Fantastic, just fantastic.
I can't imagine how long it would have taken me to figure that out on my own without such a clarification -- I was applying the same minimum hole size constraint to all holes, plus I was under the impression from experience thus far that the top hole was pretty much invariably always going to be the smallest. For some reason, on an A4 6-hole NAF with grouped holes (.75" straight bore and .1875" walls), when I was using WID to get the lowest possible target note deviations (I believe under 5% on primaries, and under 7% or so on one or two cross-fingerings) and when I was requiring all holes to have the same exact minimum hole size, I could not go above .26" without problems arising with deviations (keeping in mind it's a straight bore and I've got 14 target notes).

By allowing other holes to dip just a little below the .29 perhaps I can allow the top hole that diameter without worsening my deviations.

How you're getting less than 5% deviations on all 14 or so target notes without one notorious exception of a note or two is beyond me unless it's with stepped or graduated bore, but I've been close enough to be pretty satisfied. Now, though, I want to concentrate on purity (I like a nuance of faint, odd harmonics, but I can go back and address that after achieving purity, just as a painter demonstrates realism before advancing to style) and I want the second register to be favorable.

This has been a spectacular, eye-opening conversation for me, tying vital things together and dispelling debilitating concerns.
Sometimes I'm slower than some picking up the most technical aspects, but I'm determined.
I'd lie if I said I was always patient, but I'm determined.

Graciously enabled,
Jeff

________________________________

**Reply by edwardkort** [reply]
Jeff,

For A4 and higher flutes, I use a straight bore. Anything lower has some taper or choke in the bore. That is IMHO required to achieve a 5 cent tuning accuracy. 

Because tuning is pretty much a slam dunk, I spend most of my time refining the tone.

Cheers,
Edward
